src/Sugar.mli

```diff
@@ -144,10 +140,10 @@ val polynewtype :
 (** [polynewtype cmts pat exp] returns expression of a type-constrained
     pattern [pat] with body [exp]. e.g.:
 
-    {[
-      let f: 'r 's. 'r 's t = fun (type r) -> fun (type s) -> (e : r s t)
-    ]}
+    {v
+let f: 'r 's. 'r 's t = fun (type r) -> fun (type s) -> (e : r s t)
```

Comment:
Why do we have this diff ?

Reply:
Because this would be formatted as `let f : type r s. r s t = e`

Reply:
ok, Should we fix the indentation then ?

Reply:
done